sdk/videoanalyzer/azure-resourcemanager-videoanalyzer/README.md

```diff
@@ -1,8 +1,8 @@
-# Azure Resource Manager VideoAnalyzer client library for Java
+# Microsoft Azure SDK for Video Analyzer Management APIs
```

Comment:
The pattern for readme title should be "Azure {service-name} client library for Java"